### DISCUSSION TITLE
Different result from the models

### DISCUSSION THREAD

**Discussion by AK140901** [kyleskom/NBA-Machine-Learning-Sports-Betting#207] [Q&A] [answered]
------------------fanduel odds data------------------
Philadelphia 76ers (-124) @ Brooklyn Nets (106)
Phoenix Suns (-310) @ LA Clippers (250)
Milwaukee Bucks (-205) @ Miami Heat (172)
Memphis Grizzlies (160) @ Los Angeles Lakers (-190)
---------------XGBoost Model Predictions---------------
Brooklyn Nets vs Philadelphia 76ers (53.5%): UNDER 208.5 (73.6%)
LA Clippers (56.0%) vs Phoenix Suns: UNDER 226 (59.7%)
Miami Heat vs Milwaukee Bucks (52.6%): OVER 220.5 (50.4%)
Los Angeles Lakers vs Memphis Grizzlies (52.3%): UNDER 221 (61.8%)
--------------------Expected Value---------------------
Brooklyn Nets EV: -4.12
Philadelphia 76ers EV: -3.43
LA Clippers EV: 95.9
Phoenix Suns EV: -41.77
Miami Heat EV: 28.87
Milwaukee Bucks EV: -21.71
Los Angeles Lakers EV: -27.12
Memphis Grizzlies EV: 35.86
-------------------------------------------------------
------------Neural Network Model Predictions-----------
1/1 [==============================] - 0s 196ms/step
1/1 [==============================] - 0s 20ms/step
1/1 [==============================] - 0s 16ms/step
1/1 [==============================] - 0s 17ms/step
1/1 [==============================] - 0s 47ms/step
1/1 [==============================] - 0s 18ms/step
1/1 [==============================] - 0s 20ms/step
1/1 [==============================] - 0s 20ms/step
Brooklyn Nets (53.5%) vs Philadelphia 76ers: UNDER 208.5 (52.6%)
LA Clippers (58.2%) vs Phoenix Suns: OVER 226 (52.4%)
Miami Heat vs Milwaukee Bucks (58.0%): OVER 220.5 (53.4%)
Los Angeles Lakers (50.7%) vs Memphis Grizzlies: OVER 221 (54.2%)
--------------------Expected Value---------------------
Brooklyn Nets EV: 10.13
Philadelphia 76ers EV: -15.93
LA Clippers EV: 103.61
Phoenix Suns EV: -44.68
Miami Heat EV: 14.35
Milwaukee Bucks EV: -13.76
Los Angeles Lakers EV: -22.65
Memphis Grizzlies EV: 28.24
-------------------------------------------------------

I ran both models today and in the first game between the Brooklyn Nets and the Philadelphia 76ers, the XGBoost model gave me a different result than the NN model. Can this be correct and which model is more reliable?

**Comment by kyleskom** [comment]
Why would there be different models and expecting them to give the same results?

**Reply by AK140901** [reply]
Can you explain me where are the differents between the models?

**Comment by chriseling** [comment, marked as answer]
One it’s based on xgboost and one looks like a regular neural network
trained with back propagation

On Sat, Apr 22, 2023 at 2:17 PM AK140901 ***@***.***> wrote:

> Can you explain me where are the differents between the models?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/kyleskom/NBA-Machine-Learning-Sports-Betting/discussions/207#discussioncomment-5697070>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAVP5W72FA74L2BAQ6DK3BLXCRDGPANCNFSM6AAAAAAXHZ4HKM>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID:
> <kyleskom/NBA-Machine-Learning-Sports-Betting/repo-discussions/207/comments/5697070
> @github.com>
>
-- 
Best,
Chris

**Reply by AK140901** [reply]
which one is better, or to put it another way, which model is it where 75% on the money line is promised?

**Comment by chriseling** [comment]
Nothing is gauranteed.  The xgboost is 68.6% money line when it was
validated but it could be different now.

On Sat, Apr 22, 2023 at 11:57 PM AK140901 ***@***.***> wrote:

> which one is better, or to put it another way, which model is it where 75%
> on the money line is promised?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/kyleskom/NBA-Machine-Learning-Sports-Betting/discussions/207#discussioncomment-5698465>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAVP5W7OM7COCWYEWBG3JELXCTHERANCNFSM6AAAAAAXHZ4HKM>
> .
> You are receiving this because you commented.Message ID:
> <kyleskom/NBA-Machine-Learning-Sports-Betting/repo-discussions/207/comments/5698465
> @github.com>
>
-- 
Best,
Chris
